Mathlib/Data/Nat/Factorization/Basic.lean

```diff
@@ -122,27 +122,39 @@ theorem factorization_zero : factorization 0 = 0 := by simp [factorization]
 theorem factorization_one : factorization 1 = 0 := by simp [factorization]
 #align nat.factorization_one Nat.factorization_one
 
-/-- The support of `n.factorization` is exactly `n.factors.toFinset` -/
+/-- The `Finset` of prime divisors of `n` -/
+abbrev primeDivisors (n : ℕ) : Finset ℕ := n.factorization.support
+
 @[simp]
+theorem primeDivisors_zero : primeDivisors 0 = ∅ := by simp [primeDivisors]
+
+@[simp]
+theorem primeDivisors_one : primeDivisors 1 = ∅ := by simp [primeDivisors]
+
+/-- The support of `n.factorization` is exactly `n.factors.toFinset` -/
 theorem support_factorization {n : ℕ} : n.factorization.support = n.factors.toFinset := by
   simp [factorization]
 #align nat.support_factorization Nat.support_factorization
 
-theorem factor_iff_mem_factorization {n p : ℕ} : p ∈ n.factorization.support ↔ p ∈ n.factors := by
+theorem primeDivisors_eq_factors {n : ℕ} : n.primeDivisors = n.factors.toFinset := rfl
+
+theorem primeDivisors_eq_factorization {n : ℕ} : n.primeDivisors = n.factorization.support := rfl
+
+theorem factor_iff_mem_primeDivisors {n p : ℕ} : p ∈ n.primeDivisors ↔ p ∈ n.factors := by
```

Comment:
This name seems swapped wrt the statement. (I guess it was already wrong before your refactor, but please correct it.)